src/retrieve-validation/get-boundary.ts

```diff
@@ -0,0 +1,12 @@
+// Helper function that produces a boundary for form data
+let boundary: null | string = null;
+
+const getBoundary = (): string => {
+	if (!boundary) {
+		boundary = `----CSSValidatorBoundary${String(Math.random()).slice(2)}`;
+	}
+
+	return boundary;
```

Comment:
I can imagine some edge cases where returning the same value upon subsequent calls would cause an issue. Why did you design it this way?

Reply:
Could you describe some of those edge cases? It was designed this way to get consistent results for a random boundary threaded through various functions, the other option was passing along the randomly generated boundary as an argument, but that seemed to add a lot of complication to the design and slightly more maintenance if a change was ever made.

Reply:
Basically, it makes it no longer random each time. A user could perform a validation in their browser, sniff the network request and see the chosen boundary, and then include that boundary in a subsequent validation, thereby causing the network request to break.

Another way of putting it... this design creates an _unlikely_ boundary, not an _unpredictable_ boundary. If all we wanted was for the boundary to be _unlikely_ to be included in the CSS, we could simply hard-code an unusual string for the boundary and we wouldn't need to go to all this trouble to randomly generate the boundary on the fly. A hard-coded string would be equally as unlikely, and equally as predictable, as this design.

But that's not what we're after. What we want is for the boundary to be _unpredictable_ before the request is made.